pkg/config/config_test.go

```diff
@@ -49,6 +67,11 @@ var _ = Describe("Allocation operations", func() {
 
 	})
 
+	It("throws an error when no flat-files are found", func() {
+		_, _, err := GetFlatIPAM(true, &types.IPAMConfig{}, "")
```

Comment:
please omit the last attribute; [GetFlatIPAM](https://github.com/k8snetworkplumbingwg/whereabouts/blob/e12772b9ef977269cfabe8c079748ebeb550bb0e/pkg/config/config.go#L225) has a variadic signature :

```suggestion
		_, _, err := GetFlatIPAM(true, &types.IPAMConfig{})
```

Reply:
Sure, I missed that, oops